pkgs/top-level/all-packages.nix

```diff
@@ -11540,6 +11540,7 @@ in
   kind = callPackage ../development/tools/kind {  };
 
   khronos-ocl-icd-loader = callPackage ../development/libraries/khronos-ocl-icd-loader {  };
+  krew = callPackage ../development/tools/krew { };
```

Comment:
```suggestion

  krew = callPackage ../development/tools/krew { };
```

Reply:
I don't see what this diff is about :sweat_smile:

Reply:
You have a space at the end which fails the editor check.